src/SmartComponents/Advisor/Advisor.js

```diff
@@ -32,55 +62,92 @@ const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetch
 
     useEffect(() => {
-        recStatsStatus === 'fulfilled' && setChartData([
-            { name: intl.formatMessage(messages.critical), y: recStats.total_risk[SEVERITY_MAP.critical] },
-            { name: intl.formatMessage(messages.important), y: recStats.total_risk[SEVERITY_MAP.important] },
-            { name: intl.formatMessage(messages.moderate), y: recStats.total_risk[SEVERITY_MAP.moderate] },
-            { name: intl.formatMessage(messages.low), y: recStats.total_risk[SEVERITY_MAP.low] }
-        ]);
+        if (recStatsStatus === 'fulfilled') {
+            const { total_risk, category } = recStats;
+            setTRData([
+                { title: `${total_risk[SEVERITY_MAP.critical]} ${intl.formatMessage(messages.critical)}`, risk: SEVERITY_MAP.critical },
+                { title: `${total_risk[SEVERITY_MAP.important]} ${intl.formatMessage(messages.important)}`, risk: SEVERITY_MAP.important },
+                { title: `${total_risk[SEVERITY_MAP.moderate]} ${intl.formatMessage(messages.moderate)}`, risk: SEVERITY_MAP.moderate },
+                { title: `${total_risk[SEVERITY_MAP.low]} ${intl.formatMessage(messages.low)}`, risk: SEVERITY_MAP.low }
+            ]);
+            setCategoryData([
+                {
+                    x: intl.formatMessage(messages.availability, { count: category.Availability }), y: category.Availability,
+                    fill: colorScale[0], value: 1
+                },
+                {
+                    x: intl.formatMessage(messages.stability, { count: category.Stability }), y: category.Stability,
+                    fill: colorScale[1], value: 3
+                },
+                {
+                    x: intl.formatMessage(messages.performance, { count: category.Performance }), y: category.Performance,
+                    fill: colorScale[2], value: 4
+                },
+                {
+                    x: intl.formatMessage(messages.security, { count: category.Security }), y: category.Security,
+                    fill: colorScale[3], value: 2
+                }
+            ]);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [intl, recStats, recStatsStatus]);
 
-    const legendClick = chartData.map((data) => {
-        const risk = data.name.toLowerCase();
-        // eslint-disable-next-line max-len
-        return `${UI_BASE}/advisor/recommendations?total_risk=${SEVERITY_MAP[risk]}&reports_shown=true&impacting=true&offset=0&limit=10${selectedTags.length && '&tags=' + selectedTags.join()}`;
-    });
-
     return <TemplateCard appName='Advisor' data-ouia-safe>
-        <TemplateCardHeader title='Advisor recommendations' />
+        {advisorIncidentsStatus === 'pending' ? <Loading /> : <TemplateCardHeader titleClassName='ins-m-red'
+            title={ `${intl.formatMessage(messages.incidents, { incidents: advisorIncidents?.meta?.count })}` }>
```

Comment:
Since this will still end up on the card when `advisorIncidentsStatus === 'rejected`, should provide a default value to `incidents` in the case that `advisorIncidents?.meta?.count` evaluates to `undefined`?

maybe something like `{ incidents: advisorIncidents?.meta?.count || 0 }` ?

Reply:
yeah i was thinkin bout this believe the count will be zero not undefined, but doesn't hurt to add just in case